### PR TITLE
De38060/firefox print

### PR DESCRIPTION
**Pull request by TomMichielsD2L**
Turning the grid into a table, so that it will print..
image
Old:
[print-test-firefox.pdf](https://github.com/Brightspace/continuous-professional-development/files/4422614/print-test-firefox.pdf)
New:
[print-test-firefox2.pdf](https://github.com/Brightspace/continuous-professional-development/files/4422606/print-test-firefox2.pdf)

old chrome:
[print-test-chrome.pdf](https://github.com/Brightspace/continuous-professional-development/files/4422702/print-test-chrome.pdf)

new Chrome:
[print-test-chrome2.pdf](https://github.com/Brightspace/continuous-professional-development/files/4422705/print-test-chrome2.pdf)

### CODE REVIEW

File: src/components/cpd-record-report.js [hunk 1]
```diff
@@ -75,12 +75,34 @@ class CpdRecordReport extends BaseMixin(LitElement) {
 				font-weight: bold;
 			}
 			.record-grid {
-				display: grid;
-				grid-template-columns: 1fr 3fr;
 				border: 1px black solid;
 				page-break-before: always;
 				margin-top: 20px;
 			}
+			.record-properties {
```
Comment (jwong-d2l) [inline src/components/cpd-record-report.js:82]: is it worthwhile using the element selector instead?  td {?

File: src/components/cpd-record-report.js [hunk 2]
```diff
@@ -75,12 +75,34 @@ class CpdRecordReport extends BaseMixin(LitElement) {
...
+				padding: 0px 6px 0px 0px;
+			}
+			.property-value {
+				width: 75%;
+				padding: 0px;
+			}
+			.property-value * {
+				font-size: 0.95rem;
+			}
+			.property-name > *, .property-value > * {
```
Comment (jwong-d2l) [inline src/components/cpd-record-report.js:99]: then you could do td > *

File: src/components/cpd-record-report.js (hunk 1 above)
Comment (TomMichielsD2L) [reply src/components/cpd-record-report.js:82]: I am not a fan of element selectors :) I like being more specific. How strongly do you feel about this?

Comment (jwong-d2l) [reply src/components/cpd-record-report.js:82]: enh.. not strongly

**Pull request merged**